src/proxy-ui-api/src/test/java/org/niis/xroad/securityserver/restapi/openapi/ClientsApiControllerIntegrationTest.java

```diff
@@ -165,6 +165,11 @@ public void getAllClients() {
                 clientsApiController.findClients(null, null, null, null, null, true, false, null, false);
         assertEquals(HttpStatus.OK, response.getStatusCode());
         assertEquals(11, response.getBody().size());
+        // Test sorting order
+        List<Client> list = new ArrayList<>(response.getBody());
+        assertEquals("EE:PRO:M1:SS1", list.get(0).getId());
+        assertEquals("DUMMY:PRO:M2:SS6", list.get(5).getId());
+        assertEquals("FI:GOV:M1", list.get(10).getId());
```

Comment:
Hard-coding the expected order makes the tests brittle. Perhaps the test should verify that the ordering holds (that is, list(n) <= list(n+1)).

Reply:
The tests were updated so that they now validate the order using the related Comparator. The same Comparators are used for sorting too, but the Comparators have their own unit tests that verify that they're working correctly. In this way, the Comparators logic is tested elsewhere and the tests in this PR only validate that sorting is applied to API endpoints too.